pkg/querier/blocks_store_queryable.go

```diff
@@ -702,7 +713,7 @@ func (q *blocksStoreQuerier) fetchSeriesFromStores(ctx context.Context, sp *stor
 			// But this is an acceptable workaround for now.
 			skipChunks := sp != nil && sp.Func == "series"
 
-			req, err := createSeriesRequest(minT, maxT, convertedMatchers, skipChunks, blockIDs)
+			req, err := createSeriesRequest(minT, maxT, convertedMatchers, skipChunks, blockIDs, q.streamingChunksBatchSize)
 			if err != nil {
 				return errors.Wrapf(err, "failed to create series request")
 			}
```

Comment:
At this point the `stream` is not created. So I guess this error does not matter? Btw, I am not seeing _any_ `stream.CloseSend()` here, so maybe it is taken care somewhere and it is ok?

Reply:
>At this point the `stream` is not created. So I guess this error does not matter?

Yep, GitHub wouldn't let me put the comment any lower in the diff - I'm referring to the stream created below.

> Btw, I am not seeing any `stream.CloseSend()` here, so maybe it is taken care somewhere and it is ok?

I suspect this was taken care of by closing the context returned from `errgroup.WithContext()`, but given we can't use that context any more, we'll need to do something ourselves.

Reply:
Good catch. I am now also collecting the streams in a slice right after we get it, and if the group had any error, I am closing the streams directly instead of the stream reader (stream reader's close just closed the stream).